src/ui-test/camelk_java_debug_test.ts

```diff
@@ -0,0 +1,199 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License", destination); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+'use strict'
+
+import { assert } from 'chai';
+import path = require('path');
+import * as pjson from '../../package.json';
+import {
+	ActivityBar,
+	CustomTreeSection,
+	EditorView,
+	InputBox,
+	SideBarView,
+	TextEditor,
+	ViewItem,
+	VSBrowser,
+	Workbench
+} from 'vscode-extension-tester';
+import { Marketplace } from 'vscode-uitests-tooling';
+import { prepareEmptyTestFolder } from './utils/resourcesUtils';
+import { extensionIsActivated, sectionHasItem } from './utils/waitConditions';
+import * as uiTestConstants from './utils/uiTestConstants';
+
+const TEST_FOLDER = '../../../testFolder';
+const WORKSPACE_FOLDER = path.join(__dirname, TEST_FOLDER);
+
+const START_DEBUG_LABEL = uiTestConstants.startDebug;
+const REMOVE_INTEGRATION_LABEL = uiTestConstants.integrationRemove;
+
+describe('Test Debug on Camel K Integrations from Side Bar', function () {
+
+	before(async function () {
+		this.timeout(90000);
+		this.retries(3);
+		await VSBrowser.instance.waitForWorkbench;
+		await prepareTempWorkspaceForTests(WORKSPACE_FOLDER);
+		await VSBrowser.instance.driver.wait(camelKToolingIsEnabled);
+	});
+
+	describe('Java Debug', function () {
+		
+		const INTEGRATION_LABEL = 'java-debug-test';
+		const INTEGRATION_FILE = 'JavaDebugTest';
+
+		before(async function (){
+			this.timeout(20000);
```

Comment:
Hi @joshiraez, I recommend to increase the timeout from 20000 to 60000.
I am not confident that tasks: 
- create integration
- start integration 
with/without `hasItem` check will be always so fast.

Example:
12:09:46    Test Debug on Camel K Integrations from Side Bar
12:10:05      Java Debug
12:10:27        1) "before all" hook for "Check Java Debug available"
12:10:28      No Java Debug on Invalid Files
12:10:50        2) "before all" hook for "Test Java Debugger Not Available On Invalid File"

12:16:58    1) Test Debug on Camel K Integrations from Side Bar
12:16:58         Java Debug
12:16:58           "before all" hook for "Check Java Debug available":
12:16:58       Error: Timeout of 20000ms exceeded. For async tests and hooks, ensure "done()" is called; if returning a Promise, ensure it resolves. (/mnt/hudson_workspace/workspace/vscode/vscode-camelk-pipeline/vscode-camelk/out/src/ui-test/camelk_java_debug_test.js)
12:16:58        at listOnTimeout (node:internal/timers:559:17)
12:16:58        at processTimers (node:internal/timers:502:7)
12:16:58  
12:16:58    2) Test Debug on Camel K Integrations from Side Bar
12:16:58         No Java Debug on Invalid Files
12:16:58           "before all" hook for "Test Java Debugger Not Available On Invalid File":
12:16:58       Error: Timeout of 20000ms exceeded. For async tests and hooks, ensure "done()" is called; if returning a Promise, ensure it resolves. (/mnt/hudson_workspace/workspace/vscode/vscode-camelk-pipeline/vscode-camelk/out/src/ui-test/camelk_java_debug_test.js)
12:16:58        at listOnTimeout (node:internal/timers:559:17)
12:16:58        at processTimers (node:internal/timers:502:7)